sched/group/group_addrenv.c

```diff
@@ -46,7 +46,19 @@
  * This must only be accessed with interrupts disabled.
  */
 
-pid_t g_pid_current = INVALID_PROCESS_ID;
+pid_t g_pid_current[CONFIG_SMP_NCPUS] =
+{
+  INVALID_PROCESS_ID,
+#if CONFIG_SMP_NCPUS > 1
+  INVALID_PROCESS_ID,
+#if CONFIG_SMP_NCPUS > 2
+  INVALID_PROCESS_ID,
+#if CONFIG_SMP_NCPUS > 3
```

Comment:
can we support any number?:), do we really need to initialize it? or zero value is also good since idle thread never has the userspace map which mean use zero to identify the invalid case is also good.